app/src/ui/diff/text-diff.tsx

```diff
@@ -58,6 +57,7 @@ import * as OcticonSymbol from '../octicons/octicons.generated'
 import { WhitespaceHintPopover } from './whitespace-hint-popover'
 import { PopoverCaretPosition } from '../lib/popover'
 import { HiddenBidiCharsWarning } from './hidden-bidi-chars-warning'
+import 'codemirror/addon/scroll/simplescrollbars'
```

Comment:
I didn't have time to think this through to completion when we chatted earlier but I think with this change we should be able to substitute the `require` in `code-mirror-host` with this and remove it from here.

No need to conditionally include on darwin since we're already going to import it on both platforms here.